pkgs/applications/radio/gnuradio/default.nix

```diff
@@ -143,6 +143,10 @@ let
     };
     gr-blocks = {
       cmakeEnableFlag = "GR_BLOCKS";
+      runtime = [
+        # Required to compile wavfile blocks.
+        libsndfile
```

Comment:
libsndfile should be added to gnuradio 3.9 as well
https://www.gnuradio.org/news/2021-01-17-gnu-radio-v3.9.0.0-release/

Reply:
Good idea, thanks :)